content/en/blog/2022/rust-instrumentation/index.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Instrumenting a rust application with OTel
+linkTitle: Rust Instrumentation
+date: 2022-09-21
+author: '[Yoav Danieli](https://github.com/ydan287) (Aspecto)'
+canonical_url: https://www.aspecto.io/blog/distributed-tracing-with-opentelemetry-rust/
+---
+
+# Getting Started: OpenTelemetry in Rust
+
+![Cover image showing Rust and OpenTelemetry Logos](cover-image.webp)
+
+In this article, I will share my experience adding OpenTelemetry distributed tracing to a Rust application.
+
+I will attempt to answer the following questions:
+
+- How to instrument Opentelemetry in Rust?
+
+- How to add manual and auto instrumentations to a Rust application?
```

Comment:
I don't see any auto instrumentation. I think it doesn't even exist (yet) for rust.
```suggestion
- How to add instrumentations to a Rust application?
```